### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by holdestmade** [custom-components/places#8]
Have this issue on startup, can get round it by commenting out all refs to entity-picture in sensor.py

```
Error while setting up platform places
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 126, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/places/sensor.py", line 312, in setup_platform
    add_devices([Places(hass, devicetracker_id, name, api_key, options, home_zone, map_provider, map_zoom)])
  File "/config/custom_components/places/sensor.py", line 332, in __init__
    self._entity_picture = hass.states.get(devicetracker_id).attributes.get('entity_picture')
AttributeError: 'NoneType' object has no attribute 'attributes'
```

Maybe related to this closed issue:

https://github.com/custom-components/HomeAssistant-Places/issues/5

**Comment by jurekam** [comment]
I get the exact same error.  I've gone up to 0.96.2 with no resolution.  I'm guessing they changed the way something worked again.  It's been happening to me since 0.95.x.  Waiting on 0.96.3 to be released (version released the other day) on docker to find out it it helps.

Until then, you can at least get the component loading (without entity pics) by changing line 332 in sensor.py.  Comment out the original line so you can test later and add the second line below:

```
        # self._entity_picture = hass.states.get(devicetracker_id).attributes.get('entity_picture')
        self._entity_picture = None
```

Hope this help until they get a more permanent resolution.

**Comment by iantrich** [comment]
I think a code change in the component will be needed, not HA core

**Comment by holdestmade** [comment]
Changing the manifest.json to this (adding the device_tracker depenedency) solves this:

Thanks to @j1nx 
```
{
  "domain": "places",
  "name": "Places",
  "documentation": "https://github.com/tenly2000/HomeAssistant-Places",
  "dependencies": ["device_tracker"],
  "codeowners": ["@tenly2000"],
  "requirements": []
}
```

**Comment by iantrich** [comment]
I went ahead and added this in release 1.3. Thanks, @holdestmade. Feel free to submit PRs for these things so everyone can benefit sooner.

**Comment by jurekam** [comment]
That dependency was already in mine before my previous post (see below).  That's doesn't fix it for me.  

FYI, I just upgraded to 0.96.5 and still have the same issue.  Didn't think it would, but it was worth a shot.

`{
  "domain": "places",
  "name": "Places",
  "documentation": "https://github.com/tenly2000/HomeAssistant-Places",
  "dependencies": ["device_tracker"],
  "codeowners": ["@tenly2000"],
  "requirements": []
}`

Error:
`Traceback (most recent call last):
  File "/usr/src/app/homeassistant/helpers/entity_platform.py", line 126, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/places/sensor.py", line 312, in setup_platform
    add_devices([Places(hass, devicetracker_id, name, api_key, options, home_zone, map_provider, map_zoom)])
  File "/config/custom_components/places/sensor.py", line 332, in __init__
    self._entity_picture = hass.states.get(devicetracker_id).attributes.get('entity_picture')
AttributeError: 'NoneType' object has no attribute 'attributes'
`

**Comment by iantrich** [comment]
@jurekam I'm unable to test at the moment, but can you give this suggestion a try?
https://community.home-assistant.io/t/reverse-geocode-sensor-places-using-openstreetmap-custom-component/52230/134

**Comment by jurekam** [comment]
> @jurekam I'm unable to test at the moment, but can you give this suggestion a try?
> https://community.home-assistant.io/t/reverse-geocode-sensor-places-using-openstreetmap-custom-component/52230/134

@iantrich That works, but I figured out what caused my issue.....kind of.  If I enabled Gravatar in known_devices.yaml, this would throw the error.  If it is disabled, no error.  Maybe the entitiy_picture content is different with Gravatar enabled?  I temporarily switched local images and everything is happy again.  I've been using Gravatar images since day one, never had an issue.  Something definitely changed, but not sure what.  Not sure if this is a places component issue or HA's.  I don't remember seeing any Gravatar changes recently.  Gravatar was a great way for my family to maintain their own images (less I needed to do!).  I would like to move back to it soon.

This about the Persons component, which I'm not using yet, but it may be related:
[https://community.home-assistant.io/t/gravatar-for-person-entity/122296](https://community.home-assistant.io/t/gravatar-for-person-entity/122296)
